<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BGCMS</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter0">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="widSide" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="labSideTitle">
          <property name="text">
           <string>Collections</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="lvCollections"/>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter1">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTableView" name="lvContents">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
       <widget class="QListView" name="lvTagFilter">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::NoDragDrop</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="movement">
         <enum>QListView::Static</enum>
        </property>
        <property name="resizeMode">
         <enum>QListView::Adjust</enum>
        </property>
        <property name="layoutMode">
         <enum>QListView::Batched</enum>
        </property>
        <property name="viewMode">
         <enum>QListView::IconMode</enum>
        </property>
        <property name="selectionRectVisible">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>535</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConnect_Cms_Server"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionSetting"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuCMS">
    <property name="title">
     <string>CMS</string>
    </property>
    <widget class="QMenu" name="menuCollection">
     <property name="title">
      <string>Collection</string>
     </property>
     <addaction name="actionAdd_Collection"/>
     <addaction name="actionRename_Collection"/>
     <addaction name="actionRemove_Collection"/>
    </widget>
    <widget class="QMenu" name="menuContent">
     <property name="title">
      <string>Content</string>
     </property>
     <addaction name="actionAdd_Content"/>
     <addaction name="actionRemove_Content"/>
     <addaction name="actionRearrange"/>
     <addaction name="separator"/>
     <addaction name="actionTag_Filter"/>
    </widget>
    <addaction name="actionLogin"/>
    <addaction name="separator"/>
    <addaction name="menuCollection"/>
    <addaction name="menuContent"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCMS"/>
  </widget>
  <widget class="QToolBar" name="File">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect_Cms_Server"/>
   <addaction name="actionDisconnect"/>
   <addaction name="actionSetting"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QToolBar" name="CMS">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLogin"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_Collection"/>
   <addaction name="actionRename_Collection"/>
   <addaction name="actionRemove_Collection"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_Content"/>
   <addaction name="actionRemove_Content"/>
   <addaction name="actionRearrange"/>
   <addaction name="actionCopy_Link"/>
   <addaction name="separator"/>
   <addaction name="actionTag_Filter"/>
  </widget>
  <widget class="QStatusBar" name="CMSStatusBar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="bgcms.qrc">
     <normaloff>:/icons/icons8-Exit.png</normaloff>:/icons/icons8-Exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAdd_Content">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="bgcms.qrc">
     <normaloff>:/icons/icons8-Add File.png</normaloff>:/icons/icons8-Add File.png</iconset>
   </property>
   <property name="text">
    <string>Add Content</string>
   </property>
  </action>
  <action name="actionRemove_Content">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="bgcms.qrc">
     <normaloff>:/icons/icons8-Delete File.png</normaloff>:/icons/icons8-Delete File.png</iconset>
   </property>
   <property name="text">
    <string>Remove Content</string>
   </property>
  </action>
  <action name="actionAdd_Collection">
   <property name="icon">
    <iconset resource="bgcms.qrc">
     <normaloff>:/icons/icons8-Add Folder.png</normaloff>:/icons/icons8-Add Folder.png</iconset>
   </property>
   <property name="text">
    <string>Add Collection</string>
   </property>
  </action>
  <action name="actionRemove_Collection">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="bgcms.qrc">
     <normaloff>:/icons/icons8-Delete Folder.png</normaloff>:/icons/icons8-Delete Folder.png</iconset>
   </property>
   <property name="text">
    <string>Remove Collection</string>
   </property>
  </action>
  <action name="actionConnect_Cms_Server">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="bgcms.qrc">
     <normaloff>:/icons/icons8-Connected.png</normaloff>:/icons/icons8-Connected.png</iconset>
   </property>
   <property name="text">
    <string>Connect Cms Server</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="bgcms.qrc">
     <normaloff>:/icons/icons8-Disconnected.png</normaloff>:/icons/icons8-Disconnected.png</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionRearrange">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="bgcms.qrc">
     <normaloff>:/icons/icons8-Hashtag.png</normaloff>:/icons/icons8-Hashtag.png</iconset>
   </property>
   <property name="text">
    <string>Rearrange</string>
   </property>
   <property name="toolTip">
    <string>Rearrange</string>
   </property>
  </action>
  <action name="actionTag_Filter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="bgcms.qrc">
     <normaloff>:/icons/icons8-Filter-48.png</normaloff>:/icons/icons8-Filter-48.png</iconset>
   </property>
   <property name="text">
    <string>Tag Filter</string>
   </property>
  </action>
  <action name="actionRename_Collection">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="bgcms.qrc">
     <normaloff>:/icons/icons8-Rename Folder.png</normaloff>:/icons/icons8-Rename Folder.png</iconset>
   </property>
   <property name="text">
    <string>Rename Collection</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="icon">
    <iconset resource="bgcms.qrc">
     <normaloff>:/icons/icons8-Login-48.png</normaloff>:/icons/icons8-Login-48.png</iconset>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="icon">
    <iconset resource="bgcms.qrc">
     <normaloff>:/icons/icons8-Settings-48.png</normaloff>:/icons/icons8-Settings-48.png</iconset>
   </property>
   <property name="text">
    <string>Setting</string>
   </property>
  </action>
  <action name="actionCopy_Link">
   <property name="icon">
    <iconset resource="bgcms.qrc">
     <normaloff>:/icons/icons8-Copy Link-48.png</normaloff>:/icons/icons8-Copy Link-48.png</iconset>
   </property>
   <property name="text">
    <string>Copy Link</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lvContents</tabstop>
 </tabstops>
 <resources>
  <include location="bgcms.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionTag_Filter</sender>
   <signal>toggled(bool)</signal>
   <receiver>lvTagFilter</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
